### PR TITLE
Updated changelog

### PR DESCRIPTION
**Pull request by IvanSavenko**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -13,11 +13,16 @@
 * All in-game cheats are now case-insensitive
 * Added high-definition icon for Windows
 * Fix crash on connecting to server on FreeBSD and Flatpak builds
+* Save games now consist from a single file
```
Comment (DjWarmonger) [inline ChangeLog.md:16]: consist **of**

**Comment by dydzio0614** [comment]
include https://github.com/vcmi/vcmi/pull/2123

File: ChangeLog.md [hunk 2]
```diff
@@ -70,6 +77,7 @@
 * Improved zone placement, shape and connections
 * Improved zone passability for better gameplay
 * Improved treasure distribution and treasure values to match SoD closely
+* Navigation and water-specific spells are now banned on maps without water
```
Comment (dydzio0614) [inline ChangeLog.md:80]: this is not RMG-specific?

**Comment by dydzio0614** [comment]
Also maybe include https://github.com/vcmi/vcmi/pull/2358

File: ChangeLog.md [hunk 3]
```diff
@@ -92,10 +102,14 @@
 * Fixed highlighting of possible artifact placements when exchanging with allied hero
 * Implemented sound of flying movement (for Fly spell or Angel Wings)
 * Last symbol of entered cheat/chat message will no longer trigger hotkey
+* Right-clicking map name in scenario selection will now show file name
+* Right-clicking save game in save/load screen will now show file name and creation date
 
 ### BATTLES:
 * Implemented Tower moat (Land Mines)
 * Implemented defence reduction for units in moat
+* Implemented hero status window popup
```
Comment (dydzio0614) [inline ChangeLog.md:111]: I would name it "improved" since it worked before via right-click

**Pull request merged**